File: hw_2/primenumb.py
from datetime import datetime
import time


def getPrimeNumbers(n):
    start = datetime.now()

    result_list = []
    n = 2

    while len(result_list) < n:

        status = False;  
        result_list_numb = 0

        for result_list_numb in result_list:

            if n % result_list_numb == 0 & (n / result_list_numb != 1):
                status = True
                break


        if (status == False):
            result_list.append(n)

        n = n + 1
        
    print(datetime.now() - start)
    return result_list



if __name__ == "__main__":
    getPrimeNumbers(1)

File: hw_2/primenumb.js
function getPrimeNumbers(n) {

    const array = [];
    for (let i = 2; array.length < n; i++) {
        let status = false;
        for (let counter = 0; counter < i; counter++) {
            if (i % array[counter] === 0) {
                status = true;
                break;
            }
        }
        if (status === false) {
            array.push(i)
        };

    }
    return array;
}


console.time();
console.log(getPrimeNumbers(process.argv[2]));
console.timeEnd();